ApplicationLibCode/UserInterface/RiuWellLogTrack.cpp

```diff
@@ -126,6 +127,59 @@ class WellLogCurveInfoTextProvider : public RiuPlotCurveInfoTextProvider
 
         return "";
     }
+
+    //--------------------------------------------------------------------------------------------------
+    ///
+    //--------------------------------------------------------------------------------------------------
+    QString additionalText( RiuPlotCurve* curve, int sampleIndex ) const override
+    {
+        if ( curve )
+        {
+            std::vector<std::pair<QString, double>> propertyNameValues;
+
+            auto* sourceCurve = curve->ownerRimCurve();
+
+            if ( sourceCurve )
+            {
+                auto annotationCurves = sourceCurve->annotationCurves();
+                for ( auto annotationCurve : annotationCurves )
+                {
+                    RimDepthTrackPlot* depthTrackPlot = nullptr;
+                    annotationCurve->firstAncestorOfType( depthTrackPlot );
+                    if ( depthTrackPlot )
+                    {
+                        auto [xValue, yValue] = curve->sample( sampleIndex );
+
+                        auto depth = depthTrackPlot->depthOrientation() == RimDepthTrackPlot::DepthOrientation::VERTICAL
+                                         ? yValue
+                                         : xValue;
+
+                        auto propertyValue = annotationCurve->closestYValueForX( depth );
+
+                        // Use template to get as short label as possible. The default curve name will often
+                        // contain too much and redundant information.
+                        QString templateText = RiaDefines::namingVariableResultName();
+                        auto    resultName   = annotationCurve->annotationCurveName( templateText );
+
+                        propertyNameValues.push_back( std::make_pair( resultName, propertyValue ) );
+                    }
+                }
+            }
+
+            if ( !propertyNameValues.empty() )
```

Comment:
No need to check if `propertyNameValues` is empty: loop will also produce an empty string.